### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by bokufai**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -3,8 +3,27 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('Should return [0, 0] if age < 15', () => {
```
Comment (djtyrf312) [inline src/getHumanAge.test.js:6]: Imagine that your test fails with such text "should return [0, 0] if age < 15" how to understand exact age? Is it 14 or 10? try to use exact values and better to create constants to reuse them in the test name and expect() function

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
think about boundary values and try to make tests according to your ideas.Also check comment in the pull request and make changes

File: src/getHumanAge.test.js (hunk 1 above)
Comment (bokufai) [reply src/getHumanAge.test.js]: Fixed

**Review by Huskov-Anton — APPROVED** [review]
